### ISSUE TITLE
certificate template improvements

### ISSUE THREAD

**Issue by pdpinch** [mitodl/mitxonline#1243]
#### Acceptance Criteria:

- [ ] the alt text for the signatures is not needed, since the images are the same as the name. Best practice according to our accessibility export is to change them to `alt=""`  
- [ ] The copyright in the footer should be the same as the rest of the site, i.e. "© 2022 Massachusetts Institute of Technology". Is there a constant somewhere we can reuse?
